Building Shortcircuit XT in NixOS

I'd like to build Shortcircuit XT as an NixOS package. But the build fails for the following reason:

```
       > -- ShortcircuitXT: Using date-driven version while in alpha - 0.8.330.0
       > -- Looking for file /build/source/BUILD_VERSION
       > -- Determining version from git
       > -- Could NOT find Git (missing: GIT_EXECUTABLE)
       > CMake Error at libs/sst/sst-plugininfra/cmake/git-version-functions.cmake:5 (message):
       >   Unable to find GIT
       > Call Stack (most recent call first):
       >   libs/sst/sst-plugininfra/cmake/git-version-functions.cmake:161 (version_from_git)
       >   CMakeLists.txt:38 (version_from_versionfile_or_git)
```

CMake wants to derive some data with git, but there's no git repo in the sandbox. It could use a `BUILD_VERSION` file, but generating this file is not straightforward, since you need to also generate GIT_IMPLIED_DISPLAY_VERSION (see https://github.com/surge-synthesizer/sst-plugininfra/blob/9901d86e705e85d97c1e12a5342d6b463e26c550/cmake/git-version-functions.cmake).

It would be nice if building process was more friendly to building in a sandbox.

--- a/src/clients/clap-first/CMakeLists.txt
+++ b/src/clients/clap-first/CMakeLists.txt
@@ -64,11 +64,9 @@ if (UNIX AND NOT APPLE)
 
     # VST3 plugin (.vst3 bundle)
     if(TARGET scxt_clapfirst_vst3)
-        install(TARGETS scxt_clapfirst_vst3
-            BUNDLE  DESTINATION "${SCXT_VST3_INSTALL_DIR}"
-            LIBRARY DESTINATION "${SCXT_VST3_INSTALL_DIR}"
-            RUNTIME DESTINATION "${SCXT_VST3_INSTALL_DIR}"
-        )
+        set(v3name "$<TARGET_PROPERTY:scxt_clapfirst_vst3,LIBRARY_OUTPUT_NAME>.vst3")
+        set(v3dir "$<TARGET_PROPERTY:scxt_clapfirst_vst3,LIBRARY_OUTPUT_DIRECTORY>/../../..")
+        install(DIRECTORY "${v3dir}/${v3name}" DESTINATION "${SCXT_VST3_INSTALL_DIR}")
     endif()
 
     # Standalone app/executable

